### ISSUE TITLE
max-width to be changed 420px - Nexus 5X's width

### ISSUE THREAD

**Issue by eldenpark** [vtag/klotter-web#10]

**Comment by AlexanderEkdahl** [comment]
Should not use max-width at all. Just let it fit the screen no?

**Comment by eldenpark** [comment]
when accessed via pc web browser, if there is no max width the app gets too wide

**Comment by eldenpark** [comment]
considering the original design was based on mobile device

**Comment by AlexanderEkdahl** [comment]
I agree. It looked good when we were demoing the app. However for iPad etc it looks weird and should be using the full size.
